libs/common/src/utils/dsnpTypeConverter.ts

```diff
@@ -0,0 +1,176 @@
+import { Injectable, Logger } from '@nestjs/common';
```

Comment:
Some extra work got done in this PR which was out of scope, dumping that here for future use